docs/Deploying-PowerAuth-Standard-RESTful-API.md

```diff
@@ -1,20 +1,22 @@
-# Deploying PowerAuth Standard RESTful API Bootstrap
+# Deploying PowerAuth RESTful API
```

Comment:
Can we move this to the "Enrollment Server" documentation completely / merge it with the Docs there?

Reply:
Yes, I will move the documentation.